### DISCUSSION TITLE
Contests and Modes Disable all / Enable All

### DISCUSSION THREAD

**Discussion by resnikov** [magicbug/Cloudlog#1164] [Ideas / Feature Requests]
Would it be possible to have button to Disable All / Enable ALl in the Contests and Modes screens?

**Comment by magicbug** [comment]
This seems a reasonable request will add to the todo list

**Reply by AndreasK79** [reply]
@resnikov I added this to the code yesterday.

**Reply by resnikov** [reply]
Cheers 👍
